ext/pdo/php_pdo.h

```diff
@@ -49,17 +49,18 @@ extern zend_module_entry pdo_module_entry;
 
 PHP_MINIT_FUNCTION(pdo);
 PHP_MSHUTDOWN_FUNCTION(pdo);
+PHP_RINIT_FUNCTION(pdo);
+PHP_RSHUTDOWN_FUNCTION(pdo);
 PHP_MINFO_FUNCTION(pdo);
 
 ZEND_BEGIN_MODULE_GLOBALS(pdo)
 	zend_long  global_value;
+	HashTable  *pdo_pdbh_to_delete;
```

Comment:
Feature freeze implies no ABI/ABI changes, at least without a weighty reason and consideration. Should be to agree by RMs, just in case. Not relevant here anymore. My accent was more on mentioning the alternative way without breaching globals.

Thanks.